### PR TITLE
Use storyblok RSC client for fetching blog data in app router

### PR DESCRIPTION
**Pull request by alebedev**
## Describe your changes

- Support fetching blog data in app router.  This is a copy of original code that uses Storyblock RSC client for proper caching behavior
- Accidentally applied prettier to full project, so many one-line-changed files not related to above

## Justify why they are needed

Part of app router migration

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by alebedev** [comment]
* **#4297** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4297?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4314** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4314?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @alebedev and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/store/src/app/[locale]/cms/[[...slug]]/BlogStoryWrapper.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+import { getBlogArticles, getBlogCategories } from '@/services/storyblok/storyblok.rsc'
+import type { RoutingLocale } from '@/utils/l10n/types'
+
+type Props = {
+  locale: RoutingLocale
+  story: ISbStoryData
+  children: ReactNode
+}
+
+export async function BlogStoryWrapper({ locale, story, children }: Props) {
```
Comment (gustaveen) [inline apps/store/src/app/[locale]/cms/[[...slug]]/BlogStoryWrapper.tsx:25]: nit: should we maybe name it BlogStoryContainer?

**Review by gustaveen — APPROVED** [review]
Looks good to me!

**Comment by alebedev** [comment]
### Merge activity

* **May 30, 4:09 AM EDT**: @alebedev merged this pull request with [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4314).

**Pull request merged**